.travis.yml

```diff
@@ -0,0 +1,48 @@
+language: cpp
+
+sudo: false
+
+addons:
+  apt:
+    sources:
+      - ubuntu-toolchain-r-test
+      - george-edison55-precise-backports
+    packages:
+      - ccache
+      - cmake
+      - cmake-data
+      - libopenmpi-dev
+      - openmpi-bin
+      - gcc-5
+      - g++-5
+      - bc
+
+before_install:
+ - git -C cinch submodule init && git -C cinch submodule update
+ - git fetch --tags
+ - if [[ ${COVERAGE}  ]]; then pip install --user codecov; fi 
+
+env: #maybe add mpich later
+  global:
+    - CCACHE_CPP2=yes
+    - GVER=5
+  matrix:
+#    - MPI=ON
```

Comment:
MPI-enable build is broken for some reason, see:
<https://travis-ci.org/junghans/flecsi/jobs/112658215>
```
CMake Error at CMakeLists.txt:599 (add_test):
  add_test given unknown argument:

    mpi



-- Configuring incomplete, errors occurred!
```
This seems to be a problem in flecsi itself!